plugins/modules/ini_file.py

```diff
@@ -109,6 +109,13 @@
       - Allow option without value and without '=' symbol.
     type: bool
     default: false
+  follow:
+    description:
+    - This flag indicates that filesystem links, if they exist, should be followed.
+    - I(follow=yes) and I(state=link) can modify I(src) when combined with parameters such as I(mode).
```

Comment:
```suggestion
    - I(follow=true) and I(state=link) can modify I(src) when combined with parameters such as I(mode).
```
I'm not sure what `state=link` means though?

Reply:
oh, that was from the `copy` module. I fixed that.